docs/src/pages/en/media-captions-selectmenu.md

```diff
@@ -0,0 +1,110 @@
+---
+title: <media-captions-selectmenu> (Experimental)
+description: Experimental Media Captions Selectmenu
+layout: ../../layouts/MainLayout.astro
+source: https://github.com/muxinc/media-chrome/tree/main/src/js/experimental/media-captions-selectmenu.js
+---
+<script src="/node_modules/media-chrome/dist/experimental/media-captions-selectmenu.js" type="module"></script>
+
+A menu-button for subtitles and captions.
+
+### Default (no subtitles and captions)
+
+<media-captions-selectmenu></media-captions-selectmenu>
+
+```html
+<media-captions-selectmenu></media-captions-selectmenu>
+```
+
+### Alternate Content
+
+For alternative content for the button, slot a [media-captions-button](./media-captions-button).
+
+<media-captions-selectmenu>
+    <media-captions-button slot="button">
+      <span slot="on"><b><u>CC</u></b></span>
+      <span slot="off">CC</span>
+    </media-captions-button>
+</media-captions-selectmenu>
+
+<media-captions-selectmenu>
+    <media-captions-button slot="button" media-captions-showing="en:English%20Closed%20Captions">
+      <span slot="on"><b><u>CC</u></b></span>
+      <span slot="off">CC</span>
+    </media-captions-button>
+</media-captions-selectmenu>
+
+```html
+<media-captions-selectmenu>
+    <media-captions-button slot="button">
+      <span slot="on"><b><u>CC</u></b></span>
+      <span slot="off">CC</span>
+    </media-captions-button>
+</media-captions-selectmenu>
+
+<media-captions-selectmenu>
+    <media-captions-button slot="button" media-captions-showing="en:English%20Closed%20Captions">
+      <span slot="on"><b><u>CC</u></b></span>
+      <span slot="off">CC</span>
+    </media-captions-button>
+</media-captions-selectmenu>
+```
+
+## Styling
+
+media-captions-listbox puts all the children in its shadow DOM, which makes it a bit harder to style. One alternative is to slot a button or listbox element and then style them directly. However, this component also [exposes three parts](#exposed-parts) that can be targeted for styling.
+
+Check out this example for usage, but please try and use better colors as this color scheme is for demonstration purposes only.
```

Comment:
There's only code here, how do I see this example?

Reply:
yeah, it's broken. Let me change how I import the module and if we figure out a better way, later on, we can change it up again.

Reply:
It's fixed now.